### DISCUSSION TITLE
Drop Xamarin support

### DISCUSSION THREAD

**Discussion by thudugala** [thudugala/Plugin.LocalNotification#379] [Polls]
It is much easier to maintain the code if we drop support for Xamarin and only continue .Net Maui

**Comment by mos379** [comment]
We are not ready to migrate to Maui, maybe in half a year or so, but currently maui is just not supported by all plugins yet

**Comment by thudugala** [comment]
Xamarin support will be dropped May 1, 2024

https://dotnet.microsoft.com/en-us/platform/support/policy/xamarin
